packages/common/selectors/entities/list.js

```diff
@@ -93,13 +93,13 @@ export const personalListsByTypeSelector = createSelector(
   }),
 );
 
-export const teamListsSelector = createSelector(
+export const teamIdsSelector = createSelector(
```

Comment:
Let's rename this selector teamListsSelector.
Because in teamIdsSelector I am waiting for teamIds, but not list, which has teamId.
And this selector will return all lists 'cause in normalizr we add `teamId: entity.teamId || TEAM_TYPE.PERSONAL,`

Reply:
@cu4xxi you shouldn't wait the teamIds from the list sagas cause you don't have access to the teams state. We can add the List postfix to make the method more semantic.